piksi_tools/settings.py

```diff
@@ -266,50 +263,6 @@ def _settings_list_callback(self, sbp_msg, **metadata):
     def _settings_done_callback(self, sbp_msg, **metadata):
         self.settings_list_received = True
 
-    def _confirm_write(self, section, setting, value, retries, verbose=False):
```

Comment:
removing the confirm write means that the tool will not support v1.0, v1.1, v1.2, or v1.3 of Piksi-Multi Firmware.  Is that acceptable?